### ISSUE TITLE
Large Result Sets

### ISSUE THREAD

**Issue by reesemclean** [skabbes/uikonf_coredata_talk#2]
Are you implementing anything like faulting or batching when dealing with large result sets?

**Comment by skabbes** [comment]
Yes, batching definitely.  Faulting is a little bit too much magic.  It makes it very easy to accidentally introduce a huge latency spike on the UI.
